crates/ra_hir/src/code_model_api.rs

```diff
@@ -203,25 +215,38 @@ impl Enum {
         Ok(db.enum_data(self.def_id)?.name.clone())
     }
 
+    // TODO: deprecate in favour of variant_ids?
```

Comment:
Deprecate perhaps isn't the right word here at this stage in the proceedings - more remove!

Reply:
Right, that was the idea! ;-)